18 ReactHotLoader/webpack.config.js

```diff
@@ -14,35 +15,32 @@ module.exports = {
       extensions: ['', '.js', '.ts', '.tsx']
   },
 
+  // Toake into account: remove hot loading entry points in production
   entry: [
+    'webpack-dev-server/client?http://localhost:8080',
+    'webpack/hot/only-dev-server',
     './index.tsx',
     './css/site.css',
     '../node_modules/toastr/build/toastr.css',
     '../node_modules/bootstrap/dist/css/bootstrap.css'
   ],
 
   output: {
+    publicPath: 'http://localhost:8080/',
     path: path.join(basePath, "dist"),
     filename: 'bundle.js'
   },
 
-  //https://webpack.github.io/docs/webpack-dev-server.html#webpack-dev-server-cli
-  devServer: {
-    contentBase: './dist', //Content base
-    inline: true, //Enable watch and live reload
-    host: 'localhost',
-    port: 8080
-  },
 
   // http://webpack.github.io/docs/configuration.html#devtool
-  devtool: 'inline-source-map',
+  devtool: 'source-map',
 
   module: {
 		loaders: [
       {
         test: /\.(ts|tsx)$/,
         exclude: /node_modules/,
-        loader: 'ts-loader'
+        loaders: ['react-hot', 'babel','ts-loader']
```

Comment:
I think webpack works if we configure this line like:

`loaders: ['react-hot-loader', 'ts-loader']`

or

`loaders: ['react-hot', 'ts']`

It's the same for it